clients/python/Makefile

```diff
@@ -98,16 +102,14 @@ pylint: _check_venv_active ## runs linter (only to check errors. SEE .pylintrc e
 .PHONY: test-dev
 test-dev: _check_venv_active ## runs tests during development
 	# runs tests for development (e.g w/ pdb)
-	pytest -vv --exitfirst --failed-first --durations=10 --pdb --ignore=$(ARTIFACTS_DIR)/client $(PYTHON_DIR)
+	python -m pytest -vv --exitfirst --failed-first --durations=10 --pdb $(PYTHON_DIR)/test/test_osparc
```

Comment:
- Good move. `_check_venv_active` guarantees `python` but not `pytest` executable!
- TIP: expand options in multi-line

Reply:
will do